tests/fixtures/plotting_fixtures.py

```diff
@@ -17,17 +17,26 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
-"""Pytest configuration for plotting tests."""
-from __future__ import annotations
-
+"""Provide fixtures for non-interactive plotting tests."""
 from collections.abc import Generator
 
+import matplotlib
 import matplotlib.pyplot as plt
 import pytest
 
+# Plotting backend
+matplotlib.use('Agg')  # Non-interactive plotting
```

Comment:
This would be a bit cleaner as a fixture. Maybe in the future we might add more logic to setup test plotting. Using a `@pytest.fixture(scope='session', autouse=True)` will make sure that it'll only run once.